content/en/Platform Deep Dive/Scans/_index.md

```diff
@@ -14,70 +14,83 @@ The complexity of today's applications, combined with the constant evolution of
 
 The DAST Scanner addresses these challenges head-on. By combining the power of PtaaS with DAST, you gain a comprehensive solution for continuously assessing risk.
 
-The beta period runs until December 31, 2023. During the beta, you have access to three scans a month, allowing you to fully explore this new feature. Scans reset on the first of the month. Anticipate usage and limitation adjustments in 2024.
-
-To access the DAST Scanner Beta you must be an org owner or org member. Staff org members can also access this section to be able to better provide support.
+To get started with the DAST Scanner, you will need to add a target. Every Cobalt Customer has access to one target for free unlimited scanning. Additional targets can be purchased. Learn more about [targets](/platform-deep-dive/scans/targets).
 
+{{% alert title="Note" color="primary" %}}
 We use the following IP Address to make requests to your target: **52.19.40.38**
+{{% /alert %}}
 
 ## How to Use the DAST Scanner
 
 <!-- TODO: update this video -->
 [Watch the how-to video here.](https://videos.cobalt.io/watch/4D8kQcTmPBE3HdWy6X3fHW)
 
-1. To add a target, select the plus button
+### Add a Target
+
+- To add a [target](/platform-deep-dive/scans/targets), select the plus button
```

Comment:
I updated the screenshots and missed this line. We no longer have a plus button.
So maybe something like
```suggestion
- To add a [target](/platform-deep-dive/scans/targets), click the Add Target button.
```